app/src/main/res/values/strings.xml

```diff
@@ -1,6 +1,50 @@
 <resources>
     <string name="app_name" translatable="false">ExplURe</string>
 
+    <!-- Profile Information -->
+    <string name="user_name" translatable="false">Max Mustermann</string>
+
+    <!-- Created Routes -->
+    <string name="own_routes">Own Routes</string>
+    <string name="delete_route">DELETE ROUTE</string>
+    <string name="delete_route_message_one">"Are you sure that your route </string>
+    <string name="delete_route_message_two"> should be deleted?"</string>
+
+    <!-- Favorite Routes -->
+    <string name="favorite_routes">Favorite Routes</string>
+    <string name="remove_favorite">REMOVE FAVORITE</string>
+    <string name="remove_favorite_message_one">Are you sure that the route </string>
+    <string name="remove_favorite_message_two"> should be removed from your favorites?</string>
```

Comment:
Anstatt hier 2 Strings zu machen könntest du einfach einen [Stringplatzhalter ](https://developer.android.com/guide/topics/resources/string-resource#formatting-strings) verwenden. Z.B. 
`
<string name="remove_favorite_route_message"> Are you sure that the route %1$s should be removed from your favorites?</string>
`
Das gilt für einige andere Strings hier genauso.